### ISSUE TITLE
conditional time masking ratio can be reduced while training?

### ISSUE THREAD

**Issue by Imesh7** [k2-fsa/ZipVoice#185]
Hi,

Thanks for the code & paper,

**Background**

I have trained a model for my native language(sinhala).
Currently trained on 60 epochs.
The dataset has 12 hours dataset.
validation loss is around - 0.74 -0.76 
But still can't pronounce any word at all. BTW epoch 60's model is better than the 20 epoch's sampling output.

**My question is**
Have you guys ever tried out by change/reduced the `conditional_time_masking` ratio.

Currently the ratio is like 0.7-1.0 range.(hardcoded in the code)

https://github.com/k2-fsa/ZipVoice/blob/2f7326fbfe999a3ad179e3f1af82a424d4a62819/zipvoice/models/zipvoice.py#L358-L361

Some LLMs suggesting me reduce it to like as below,

1. "Reduce masking to 30-50%"
2. or may be I am lack of training time?(Train some more)

Any idea/suggestions about this.Much appreciated.

Thanks

**Comment by zhu-han** [comment]
The number of training epochs is too small for a dataset of this size. You could try using around 1000 epochs.
